service/src/main/resources/api/service_openapi.yaml

```diff
@@ -302,6 +302,97 @@ paths:
         500:
           $ref: '#/components/responses/ServerError'
 
+  "/v2/studies/{studyId}/cohorts":
+    parameters:
+      - $ref: '#/components/parameters/StudyIdV2'
```

Comment:
It's defined under components > parameters, but it doesn't show up in this PR review because I added it when I implemented the study endpoints. I'm completely on board with a name change for study, but let's do it in a separate PR, because all the internal classes should change also. `Collection` sounds good to me -- let's run it by the PMs first though.